src/languages/es.ts

```diff
@@ -2721,6 +2721,18 @@ export default {
             errorDescriptionPartTwo: 'contacta con el conserje',
             errorDescriptionPartThree: 'por ayuda.',
         },
+        restrictedAction: {
+            restricted: 'Restringido',
+            expensesAreCurrentlyRestricted: ({ workspaceName }) => `Los gastos para ${workspaceName} están actualmente restringidos`,
+            pleaseReachOutToYourWorkspaceAdmin: 'Si tienes alguna pregunta, ponte en contacto con el administrador de su espacio de trabajo.',
+            chatWithYourAdmin: 'Chatea con tu administrador',
```

Comment:
NAB: can you move pleaseReachOutToYourWorkspaceAdmin and chatWithYourAdmin translations below addPaymentCardToContinueUsingWorkspace? It looks better if the translations are in the same order